pkg/graveler/ref/manager_test.go

```diff
@@ -150,6 +169,17 @@ func TestManager_DeleteRepository(t *testing.T) {
 			if !errors.Is(err, graveler.ErrRepositoryNotFound) {
 				t.Fatalf("expected ErrRepositoryNotFound, got: %v", err)
 			}
+
+			// Create after delete
+			testutil.Must(t, tt.refManager.CreateRepository(ctx, repoID, graveler.Repository{
+				StorageNamespace: "s3://foo",
+				CreationDate:     time.Now(),
+				DefaultBranchID:  "weird-branch",
+			}))
+			_, err = tt.refManager.GetRepository(context.Background(), "example-repo")
```

Comment:
I would validate the returned repo, at least that the unique identifier is different that the one of the deleted repo.

Reply:
Creation of instance ID is in the graveler scope not in ref manager scope - unless you want to change that, this check is irrelevant